src/shared/ui/survey/StackedCheckboxItem/StackedCheckboxItem.tsx

```diff
@@ -83,7 +83,11 @@ const StackedCheckboxItem: FC<Props> = ({
         options={memoizedOptions}
         renderCell={(index, option) => {
           return (
-            <YStack onPress={() => onValueChange(option, index)}>
+            <YStack
+              px="$4"
+              py="$3"
+              onPress={() => onValueChange(option, index)}
+            >
```

Comment:
Please use [hitSlope](https://reactnative.dev/docs/pressable#hitslop) property instead. Also, I'd recommend making changes in checkboxes/radios components instead of stacked items